package.json

```diff
@@ -98,6 +98,7 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.2.5",
+    "nodemon": "^1.9.1",
```

Comment:
> you must explicitly tell npm to install globally as nodemon is a command line utility.

@rkmax I think that it will be better to remove this dependency.
@nathanmarks Do we need to add it to our documentation?